### PR TITLE
Added keyboardSpace parameter to toggle state

### PR DESCRIPTION
**Pull request by danvass**

### CODE REVIEW

File: KeyboardSpacer.js [hunk 1]
```diff
@@ -59,17 +59,19 @@ class KeyboardSpacer extends React.Component {
...
-        }, () => ('onToggle' in this.props ? this.props.onToggle(true) : null));
+        }, () => ('onToggle' in this.props ? this.props.onToggle(true, keyboardSpace) : null));
     }
 
     resetKeyboardSpace() {
         this.setState({
             keyboardSpace: 0,
             isKeyboardOpened: false
-        }, () => ('onToggle' in this.props ? this.props.onToggle(false) : null));
+        }, () => ('onToggle' in this.props ? this.props.onToggle(false, keyboardSpace) : null));
```
Comment (Andr3wHur5t) [inline KeyboardSpacer.js]: `keyboardSpace` is never defined in this context

**Comment by danvass** [comment]
@Andr3wHur5t Thanks didn't copy it over properly, fixed.

**Comment by Andr3wHur5t** [comment]
Code looks good, would you like to add your docs to this PR as well?

**Comment by danvass** [comment]
@Andr3wHur5t done.

**Comment by Andr3wHur5t** [comment]
:+1:

**Comment by Andr3wHur5t** [comment]
Published to NPM

**Pull request merged**